File: src/com/gmail/marcosav2010/connection/exception/ConnectionException.java
package com.gmail.marcosav2010.connection.exception;

public class ConnectionException extends Exception {

	private static final long serialVersionUID = 4764600295741975224L;

	public ConnectionException(String msg) {
		super(msg);
	}
	
	public ConnectionException(String msg, Throwable t) {
		super(msg, t);
	}
}

File: core/src/main/java/com/gmail/marcosav2010/connection/exception/ConnectionException.kt
package com.gmail.marcosav2010.connection.exception

class ConnectionException : Exception {

    constructor(msg: String) : super(msg)
    constructor(msg: String, t: Throwable) : super(msg, t)

    companion object {
        private const val serialVersionUID = 4764600295741975224L
    }
}